openssl/src/ssl/mod.rs

```diff
@@ -3104,6 +3104,17 @@ impl SslRef {
             }
         }
     }
+    #[corresponds(SSL_add1_chain_cert)]
+    #[cfg(ossl102)]
+    pub fn add_chain_cert_pem(&mut self, chain: &[u8]) -> Result<(), ErrorStack> {
```

Comment:
`SSL_add1_chain_cert` doesn't take a byte buffer, it takes an `X509` object: https://www.openssl.org/docs/man1.1.1/man3/SSL_add1_chain_cert.html.

Please add a test.

Reply:
done